solr/core/src/java/org/apache/solr/update/SolrIndexConfig.java

```diff
@@ -87,6 +100,7 @@ private SolrIndexConfig(SolrConfig solrConfig) {
     maxBufferedDocs = -1;
     ramBufferSizeMB = 100;
     ramPerThreadHardLimitMB = -1;
+    maxCommitMergeWaitMillis = -1;
```

Comment:
Lets use `org.apache.lucene.index.IndexWriterConfig#DEFAULT_MAX_COMMIT_MERGE_WAIT_MILLIS`.  In the future, this constant might change, and that's good.

Reply:
I don't think that's a good idea. "-1" from the Solr perspective means "don't set the value".  If someone has a custom merge policy where they have a different value set by code, we would be changing it to `org.apache.lucene.index.IndexWriterConfig#DEFAULT_MAX_COMMIT_MERGE_WAIT_MILLIS` without them having set anything on `solrconfig.xml`. 
> In the future, this constant might change, and that's good.

That's fine, since we aren't calling `iwc.setMaxCommitMergeWaitMillis(...)` in the default case, we'll be taking the new default for all cores where no alternative configuration has been provided.

Reply:
ok, good point.